CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.46.2] - 2023-09-26
+- Fix ByteBuffer errors
```

Comment:
Let's be more specific about this. I suggest:

> Fix usage of `ByteBuffer` to be compatible with Java 8 runtimes.

Reply:
Updated.